web6/web6.go

```diff
@@ -383,7 +120,7 @@ func SetCookies(cookie_map map[string]interface{}, w http.ResponseWriter, r *htt
 		new_cookie.Path = "/"
 		http.SetCookie(w, &new_cookie)
 
-		UdnLogLevel(nil, log_info, "** Setting COOKIE: %s = %s\n", key, value)
+		yudiencore.UdnLogLevel(nil, log_info, "** Setting COOKIE: %s = %s\n", key, value)
```

Comment:
This shouldnt be prefixed every time.  It's too much work to type, it will cause cheats like fmt.Printf().

This should still be top-level function.

Reply:
This makes it way easier to debug and see where the function is being called from. Otherwise I have to search many different imports to see which one.

Reply:
alternatively I can alias it like, `yc.UdnLogLevel(...)` I can still reference imports to see which package it's coming from